content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+under a GPL3+ [copyleft](https://en.wikipedia.org/wiki/Copyleft) license.
+
+This policy is unfulfilled if your artifacts contain one or more packages with
+a violating license.
+
+## View policy status
+
+The **Overview** tab of the [Docker Scout Dashboard](https://scout.docker.com/)
+displays a summary of recent changes in policy for your repositories.
+This summary shows images that have seen the most change in their policy
+evaluation between the most recently analyzed tag and the previous tag.
+
+![Policy overview](../images/policy-overview.png)
+
+### Policy status per repository
+
+The **Images** tab shows the current policy status, and recent policy trend,
+for all images in the selected environment. The **Policy status** column in the
+list shows:
+
+- Number of fulfilled policies versus the total number of policies
+- Recent policy trends
+
+![Policy status in the image list](../images/policy-image-list.png)
+
+The policy trend, denoted by the directional arrows, indicates whether a tag is
+better, worse, or unchanged in terms of policy, compared to the previous tag in
+the same environment.
+
+- The green arrow pointing upwards shows the number of policies that got better
+  in the latest version of this image.
+- The red arrow pointing downards shows the number of policies that got worse
+  in the latest version of this image.
+- The gray arrow pointing sideways shows the number of policies that were
+  unchanged in the latest version of this image.
+
+If you select a repository, you can open the **Policy** tab for a detailed
+description for the policy delta for the most recently analyzed tag and its
+predecessor.
+
+### Detailed evaluation results
+
+To view the full results of a policy evaluation for a tag, select the tag in
+the Docker Scout Dashboard and open the **Policy** tab. This shows a breakdown
+for all policy violations for the current tag.
+
+![Detailed tag evaluation results](../images/policy-detailed-results.png)
+
+### Policy change by external factors
+
+Sometimes, the **Policy status** column in the **Images** tab displays an
+additional warning or informational icon.
+
+![Icons next to policy status](../images/policy-status-icons.png)
+
+Policy evaluation results are not static. Policy status for a tag can change if
+new information becomes available, or if your policy configuration changes.
+These icons appear when the policy status for a tag changed due to external
+factors.
+
+## Remediate policy violations
+
+To remediate issues in your artifacts that cause policy violations, you can use
+the tag view in the Docker Scout Dashboard.
+
+1. Go to the **Images** view in the Docker Scout Dashboard.
+2. In the list of repositories, select the image that you want to improve.
+3. Select an image tag.
+4. Open the **Policy** tab.
+
+The **Policy** tab shows you all of the policy violations for the current tag.
+Expand the policy menus to learn which packages in the image that are causing
```

Comment:
there's an excess "that" here....should be "...which packages in the image are causing..."